infrastructure/base/vars/terraform.tfvars

```diff
@@ -6,7 +6,7 @@ project            = "afoco"
 production_domain = "climation.afocosec.org"
 staging_domain    = "climation-staging.afocosec.org"
 
-beanstalk_platform = "64bit Amazon Linux 2 v3.6.0 running Docker"
+beanstalk_platform = "64bit Amazon Linux 2023 v4.0.0 running Docker"
```

Comment:
hmm, maybe that is reason why different nginx setup is required? Because of different platform?

Reply:
nope, see the AWS link